openassessment/workflow/migrations/0002_auto_20170724_1537.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
It's the same thing that's happening [here](https://github.com/edx/edx-submissions/pull/65/files#r130444360) - We can remove the `django_extensions` dependency fairly easily, but that touches a field on a model, which leads to a migration. Said migration *should* be a no-op, but there are some pending operational questions to sort out.

The migration should have been in the last commit; I had forgotten that it needed to be generated and ran into a "please run makemigrations" warning while fixing other things. I'll work out the details with Kevin on my other PR and update this one with the results of that conversation.